content/docs/gherkin/step-organization.md

```diff
@@ -124,6 +126,6 @@ This helps in a number of ways,
 
 You can handle other behaviours, like *validating a webpage, clicking a button, etc.*, the same way.
 
-We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.
+{{% text "java" %}}We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.{{% /text %}}{{% text "kotlin" %}}We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.{{% /text %}}
```

Comment:
Why these conditionals? Do you not think the paragraph is relevant for all readers?

Reply:
Exactly. As mentioned in the commit message (which I realize isn’t that noticeable), the Factory pattern is not terribly useful in Ruby or JS.